src/mem/ruby/slicc_interface/Message.hh

```diff
@@ -126,6 +127,9 @@ class Message
   protected:
     int m_block_size = 0;
 
+    // Needed to call MacheinType_base_count/level
+    const RubySystem *p_ruby_system = nullptr;
```

Comment:
As far as I can tell, this is *unused*. I grepped for it and I could only find the following occurences:

```
src/mem/ruby/slicc_interface/Message.hh:          p_ruby_system(rs)
src/mem/ruby/slicc_interface/Message.hh:    const RubySystem *p_ruby_system = nullptr;
```

Can we just delete it?

Reply:
I'm pretty sure it's used. Delete that line and it won't compile. I'm certain I checked this.

Reply:
It works for me assuming I remove both lines. I imagine it worked to move it to "protected" because the compiler can't guarantee that a subclass doesn't use it.

Reply:
I stand corrected. I don't know why I had that assumption originally. The latest push to the PR has this line removed.